tutorials/source-modeling/plot_source_alignment.py

```diff
@@ -97,30 +99,51 @@
 # .. role:: green
 # .. role:: red
 #
-# 1. Neuromag head coordinate frame ("head", :pink:`pink axes`)
-#      Defined by the intersection of 1) the line between the LPA
-#      (:red:`red sphere`) and RPA (:purple:`purple sphere`), and
-#      2) the line perpendicular to this LPA-RPA line one that goes through
-#      the Nasion (:green:`green sphere`).
-#      The axes are oriented as **X** origin→RPA, **Y** origin→Nasion,
+# 1. Neuromag/Elekta/MEGIN head coordinate frame ("head", :pink:`pink axes`)
+#      The head coordinate frame is defined through the coordinates of
+#      anatomical landmarks on the subject's head: Usually the Nasion (`NAS`_),
+#      and the left and right preauricular points (`LPA`_ and `RPA`_).
+#
+#      Concretely, the head coordinate frame is defined by the intersection of
+#
+#      1. the line between the LPA (:red:`red sphere`) and RPA
+#         (:purple:`purple sphere`), and
+#      2. the line perpendicular to this LPA-RPA line one that goes through
+#         the Nasion (:green:`green sphere`).
+#
+#      The axes are oriented as **X** origin→RPA, **Y** origin→NAS,
 #      **Z** origin→upward (orthogonal to X and Y).
 #
-#      .. note:: This gets defined during the head digitization stage during
-#                acquisition, often by use of a Polhemus or other digitizer.
+#      .. note:: The required 3D coordinates for defining the head coordinate
+#                frame (NAS, LPA, RPA) are measured at a stage separate from
+#                the MEG data recording. There exist numerous devices to
+#                perform such measurements, usually called "digitizers". For
+#                example, see the devices by the company `Polhemus`_.
+#
+#      .. note:: Other MEG manufacturers may have different definitions of the
+#                coordinate head frame, a good overview can be seen in the
+#                section "How are the different head and MRI coordinate systems
+#                defined?" in the FAQ of the FieldTrip toolbox WIKI.
 #
 # 2. MEG device coordinate frame ("meg", :blue:`blue axes`)
-#      This is defined by the MEG manufacturers. From the Elekta user manual:
+#      The MEG device coordinate frame is defined by the respective MEG
+#      manufacturers. All MEG data is acquired with respect to this coordinate
+#      frame. To account for the anatomy and position of the subject's head, we
```

Comment:
anatomy?

Reply:
maybe some people have a wider head than others, i.e., more distance between LPA and RPA?

Reply:
yes but how does the HPI coil factor in there?